airbyte-integrations/connectors/source-freshdesk/source_freshdesk/api.py

```diff
@@ -114,7 +115,6 @@ class StreamAPI(ABC):
     """Basic stream API that allows to iterate over entities"""
 
     result_return_limit = 100  # maximum value
-    maximum_page = 500  # see https://developers.freshdesk.com/api/#best_practices
```

Comment:
so there is no need for the export endpoint?

Reply:
@sherifnada Yes. Reading pages with` page_number > 800` works fine.